### PR TITLE
Rule for Advanced IP/Port Scanner update check

### PR DESCRIPTION
**Pull request by Tomasuh**
Rule to detect Advanced IP/Port Scanner update check

- http://www.advanced-port-scanner[.]com/checkupdate.php?lng=en&ver=2-5-3680&beta=n&type=upd&rmode=p&product=aps 
- http://www.advanced-ip-scanner[.]com/checkupdate.php?lng=en&ver=2-5-3499&beta=n&type=upd&rmode=p&product=aips

I avoided matching towards domain name as those appear more likely to change/expand.

This is my first rule created from scratch so any feedback is welcome!

### CODE REVIEW

**Comment by Neo23x0** [comment]
I wouldn't reference to samples on Virustotal but to the the official page: 
https://www.advanced-ip-scanner.com/

**Comment by Neo23x0** [comment]
I wouldn't use that MITRE ATTACK category and miss at least a single technique 

e.g. 
https://attack.mitre.org/techniques/T1590/

and the category is `reconnaissance`

File: rules/proxy/advanced_ip_and_port_scanner_upd_check.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+date: 2022/08/14
+logsource:
+    category: proxy
+detection:
+    selection:
+      # Example request: http://www.advanced-port-scanner.com/checkupdate.php?lng=en&ver=2-5-3680&beta=n&type=upd&rmode=p&product=aps 
+      # Example request2: http://www.advanced-ip-scanner.com/checkupdate.php?lng=en&ver=2-5-3499&beta=n&type=upd&rmode=p&product=aips 
+      c-uri|contains: '/checkupdate.php?'
+      c-uri-query|contains|all:
+        - 'lng='
```
Comment (Neo23x0) [inline rules/proxy/advanced_ip_and_port_scanner_upd_check.yml:20]: The indentation mixes 4 and 2 spaces

File: rules/proxy/advanced_ip_and_port_scanner_upd_check.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        - 'ver='
+        - 'beta='
+        - 'type='
+        - 'rmode='
+        - 'product='
+    condition: selection
+fields:
+  - c-uri
```
Comment (Neo23x0) [inline rules/proxy/advanced_ip_and_port_scanner_upd_check.yml:28]: The c-uri field isn't the one relevant for further investigations. It's `c-ip` the client machine from which the request was sent.

**Comment by Tomasuh** [comment]
Thank you for the feedback @Neo23x0 , the improvements you suggested should be implemented + escaping the ? character and rename the file to start with proxy_

**Review by Neo23x0 — APPROVED** [review]

**Pull request merged**